src/strophe.js

```diff
@@ -10,5 +10,6 @@ global.$iq = strophe.default.$iq;
 global.$msg = strophe.default.$msg;
 global.$pres = strophe.default.$pres;
 global.Strophe = strophe.default.Strophe;
+global.b64_sha1 = strophe.default.b64_sha1;
```

Comment:
Maybe it's not mandatory to put it in global too (the export line 15 is sufficient for strophejs-plugin-caps to work), but I just followed what has been done with other exports. We could remove this line change.